user_docs/en/changes.t2t

```diff
@@ -36,7 +36,7 @@ Note:
 - Added commands for toggling multiple modifiers simultaneously with a Braille display (#13152)
 - The Speech Dictionary dialog now features a "Remove all" button to help clear a whole dictionary. (#11802)
 - Added support for Windows 11 Calculator. (#13212)
-- In Microsoft Word with UI Automation enabled on Windows 11, line numbers, section numbers and layout column numbers can now be reported. (#13283)
+- In Microsoft Word with UI Automation enabled on Windows 11 section numbers can now be reported. (#13283, #13515)

```

Comment:
Only column numbers were removed. Line numbers are still supported if the user turns them on.